build.sbt

```diff
@@ -27,23 +27,36 @@ val commonSettings = Seq(
   description := "ML Aggregators",
   scalaVersion := "2.11.12",
   crossScalaVersions := Seq("2.11.12", "2.12.5"),
-  scalacOptions ++= Seq("-target:jvm-1.8", "-deprecation", "-feature", "-unchecked"),
+  scalacOptions ++= Seq("-target:jvm-1.8",
+                        "-deprecation",
+                        "-feature",
+                        "-unchecked"),
   javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint:unchecked"),
-  javacOptions in (Compile, doc)  := Seq("-source", "1.8"),
-  publishTo := Some(if (isSnapshot.value) Opts.resolver.sonatypeSnapshots else Opts.resolver.sonatypeStaging),
-  releaseCrossBuild             := true,
+  javacOptions in (Compile, doc) := Seq("-source", "1.8"),
+  scalafmtSbtCheck := true,
```

Comment:
fails if code is not properly formatted ... user needs to format the code manually.

not sure if we should go with this or with `scalafmtOnCompile`.